packages/components/src/styles/mixins/_interactive-dark-theme.scss

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+ @use "../mixins/focus-ring" as *;
+
+ // TODO: Deprecate SideNav specific dark theme tokens in favor of general dark theme tokens
+
+// Used to apply dark theme to interactive elements such as Button & Dropdown
+@mixin hds-interactive-dark-theme($add-visible-border: true, $is-within-side-nav: false) {
+  @if $is-within-side-nav {
+    // sidenav specific dark theme tokens
+    color: var(--token-side-nav-color-foreground-strong);
+    background-color: transparent; // test
+    border-radius: var(--token-side-nav-body-list-item-border-radius);
+  } @else {
+    // general dark theme tokens
+    color: var(--token-color-palette-neutral-0);
+    background-color: var(--token-color-palette-neutral-700);
+    border-radius: var(--token-form-control-border-radius);
```

Comment:
technically it's not the `form-control-border-radius` but something like `button-border-radius`; unfortunately we don't have such token, only a variable `$hds-button-border-radius` in the `mixins/_button.scss`, maybe you can import the mixin and use that variable instead? (I've never tried, but probably should work)

Reply:
Ok, will see. (I picked that one because it seemed closest.)